lib/src/file_util.rs

```diff
@@ -146,6 +162,51 @@ pub fn persist_content_addressed_temp_file<P: AsRef<Path>>(
     }
 }
 
+/// Symlinks are always available on UNIX, but on Windows they may not be
+/// enabled
```

Comment:
nit: instead of duplicating this description, i would put only "Symlinks are always available on UNIX" here and then below a short description of when they are available on windows

Reply:
done